lib/assistant.js

```diff
@@ -410,7 +441,7 @@ class Assistant {
 
     // #region Admin Endpoints & HTTP Handlers
 
-    initAdminEndpoints (RED, { inlineCompletionsEnabled }) {
+    initAdminEndpoints (RED) {
```

Comment:
`inlineCompletionsEnabled` param has been removed from the signature but it is still passed in the call at line 177.